docs/commands/preview.md

```diff
@@ -0,0 +1,62 @@
+# `preview`
+
+## Introduction
+
+The `preview` command starts a preview server for a Redocly project.
+
+## Usage
+
+```bash
+redocly preview
+redocly preview --product=revel
+redocly preview --product=reef --plan=pro
+redocly preview --product=reef --plan=pro --source-dir=./my-docs-project --port=4001
+```
+
+## Options
+
+| Option       | Type    | Description                                                                                                                                                                           |
+| ------------ | ------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| --help       | boolean | Shows help.                                                                                                                                                                           |
+| --plan       | string  | Product plan to use in preview. <br/> **Possible values:** `pro`, `enterprise`. The default value is `enterprise`.                                                                    |
+| --product    | string  | Name of a project to preview the product with. <br/> **Possible values:** `redoc`, `revel`, `reef`, `realm`, `redoc-revel`, `redoc-reef`, `revel-reef`. The default value is `realm`. |
+| --source-dir | string  | Path to the project directory. The default value is `.` (current directory).                                                                                                          |
+| --port       | number  | The port to run the preview server on. The default value is `4000`.                                                                                                                   |
+| --version    | boolean | Shows version number.                                                                                                                                                                 |
+
+## Examples
+
+### Select a product for preview
+
+Instead of inferring the product package to use for preview from `package.json` or using Realm by default, you can specify the product package as an argument.
```

Comment:
To make the wording more positive, start with "Specify the package to use ...." and then explain that if the argument isn't supplied, we'll try to guess.